sdk/web-pubsub/web-pubsub/src/hubClient.ts

```diff
@@ -748,4 +803,18 @@ export class WebPubSubServiceClient {
       span.end();
     }
   }
+
+  public async generateClientToken(options: GenerateClientTokenOptions = {}) {
```

Comment:
To confirm: this method works regardless of your auth strategy?

Reply:
I don't believe it's intended to, but we are confirming with service team and may reshape this a bit depending on what we hear.